feeds-redlist.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "url": "http://nashutzu.blogspot.com/feeds/posts/default",
+    "lastUpdate": 0
```

Comment:
Sometimes you use a `Number` and others a Date serialized to a `String`.  I'd suggest `null` if you mean "not set".

Reply:
Good point! I'll have it write a null for unresponsive/ dead feeds.